kotlin-source/src/main/kotlin/com/example/flow/ExampleFlow.kt

```diff
@@ -64,89 +76,46 @@ object ExampleFlow {
             // Stage 1.
             progressTracker.currentStep = GENERATING_TRANSACTION
             // Generate an unsigned transaction.
-            val txCommand = Command(IOUContract.Commands.Create(), iou.participants.map { it.owningKey })
-            val unsignedTx = TransactionType.General.Builder(notary).withItems(iou, txCommand)
+            val iouState = IOUState(IOU(iouValue), serviceHub.myInfo.legalIdentity, otherParty)
+            val txCommand = Command(IOUContract.Commands.Create(), iouState.participants.map { it.owningKey })
+            val txBuilder = TransactionType.General.Builder(notary).withItems(iouState, txCommand)
```

Comment:
Done. Can't do the same in Java, because of the default params. I'll put in a PR for an overload.